front-end/src/host/HostApp.tsx

```diff
@@ -49,6 +50,12 @@ export default function HostApp(props: IHostProps) {
 
   const [loaded, setLoaded] = React.useState<boolean>(false);
   const [muted, setMuted] = React.useState<boolean>(false);
+
+  const [announcementAudioObjects, setAnnouncementAudioObjects] = React.useState<any>([]);
```

Comment:
Create state on `<HostApp />` for the array of announcements. Announcements can be added through context, and once added, they will automatically play because the `<HostAnnouncementQueue />` element re-renders.